src/store/modules/dashboard/type.ts

```diff
@@ -45,5 +45,6 @@ export interface DomainDashboardItem extends DashboardItem {
 }
 export interface ProjectDashboardItem extends DashboardItem {
     project_dashboard_id: string;
+    project_id: string;
```

Comment:
question: Are these properties are the spec of backend data? I mean, `ProjectDashboardItem` is used for backend api parameter only?
If it is not, please follow the convention rule. 
We use camel case for every javascript variable, object property names.